internal/indexmanagement/scripts.go

```diff
@@ -50,6 +51,31 @@ def deleteIndices(index):
     sys.stdout = original_stdout
     return False
 
+def deleteByQuery(index, namespaceSpecs, defaultAge):
+  original_stdout = sys.stdout
+  try:
+    es_client = getEsClient()
+    s = Search(using=es_client)
+
+    #Extract string values of namespace and minAge from namespaceSpecs to feed into dsl-query
+    namespaceSpecs = json.loads(namespaceSpecs)
+    for namespaceName in namespaceSpecs:
+      s = s.query('prefix', kubernetes__namespace_name=namespaceName)
+      minAge = namespaceSpecs[namespaceName]
+      if minAge == "":
+        s = s.filter('range', **{'@timestamp': {'lt': 'now-{}'.format(defaultAge)}})
+      else:
+        s = s.filter('range', **{'@timestamp': {'lt': 'now-{}'.format(minAge)}})
+      response = es_client.delete_by_query(index=index, body=s.to_dict(), doc_type="_doc")
+      #print (json.dumps(s.to_dict(), indent=2, sort_keys=True))
+      print (json.dumps(response, indent=2))
```

Comment:
Need to drop these debug lines

Reply:
I think I will keep this one:
`print (json.dumps(response, indent=2))`
as it shows the detailed response and how many documents were delete or were in conflict

Reply:
But except a developer no one would be interested in this information right? WDYT?
If it can be printed in higher log level then it makes sense to keep here.